### ISSUE TITLE
envelope

### ISSUE THREAD

**Issue by JunenuJ** [fuzhenfz/Calculate-Temporal-and-Spectral-Envelope-of-Vowel#1]
hi，I want to confirm the details of figure shown  this code. The envelope means spectral envelope ?

**Comment by fuzhenfz** [comment]
> hi，I want to confirm the details of figure shown this code. The envelope means spectral envelope ?

It depends.
TemporalEnvelope.m gives you the temporal envelope, or named as the acoustical envelope. It's a function of time. 
SpectralEnvelop.m gives you the spectral envelope. It's a function of frequency.

**Comment by JunenuJ** [comment]
Thank you. I understand this function.I try to plot spectral envelope about my wav  and find that it cannot show the smoothed envelope well. The spectrum envelope is always displayed below the formant

**Comment by fuzhenfz** [comment]
> Thank you. I understand this function.I try to plot spectral envelope about my wav and find that it cannot show the smoothed envelope well. The spectrum envelope is always displayed below the formant

Yes, this implementation could have this issue, but I think it could give you the general trend. I attach one of my examples here, so you can compare it with yours. 
![a3](https://user-images.githubusercontent.com/12341414/90246274-d7656b80-de66-11ea-9f14-850577b475bf.jpg)
